homeassistant/components/reolink/services.yaml

```diff
@@ -16,3 +16,29 @@ ptz_move:
           min: 1
           max: 64
           step: 1
+
+chime_play:
+  target:
+    device:
+      integration: reolink
+      model: "Reolink Chime"
+    entity:
+      integration: reolink
+      domain: select
```

Comment:
Why do we still allow entities to be selected? Is the device alone not enough?

Reply:
You can always select entities when using the target, I am merely filtering.
And I did make it work through entities or devices, both work.
That keeps me happy since I can use the entity id :)

And you can also use device_id is you prefer.

Reply:
It's not up to each integration to add other ids than the required id, just for the sake of ease. That should be handled centrally by Home Assistant if we want to do that.

Please remove entity_id selector.

Reply:
Sure I will remove it, but it does not make any sense to me.
Then you get a target select in the UI in which the user can select devices or entities (or area's or even labels) and when the user selects a entity it will just not do anything.

Nothing in the UI tell the user to use the device as target and not a entity as target.
Seems very confusing to me.

Reply:
entity selection has been removed